### DISCUSSION TITLE
installing azuracast on a vps along with webmin

### DISCUSSION THREAD

**Discussion by ElvisGarces** [AzuraCast/AzuraCast#6605] [General]
Has anyone tried installing azuracast on a vps along with webmin?

If someone has already done it, could they tell us how it went, how both environments work, and how they coexist on the same server.

Remember that Webmin is a web hosting manager that, like azuracast, is free software and some, if not many of us who have online radio need to host the radio's website, but that is expensive and we are already paying for a vps like that. It would be important to be able to use it for that dual purpose.

**Comment by Zagrthos** [comment]
You can install it through docker and then install Webmin? I don't see any problem there
